catalog/ui/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.hbs

```diff
@@ -18,6 +18,14 @@
         Run
     </div>
 </div>
+<div class="query-interaction interaction-hit-count" title="Hit Count" data-help="Executes the search returning only the hit count">
+    <div class="interaction-icon">
+        <b>#</b>
+    </div>
+    <div class="interaction-text">
+        Hit Count
```

Comment:
✏️ All the other options in this menu start with "verb" actions (`Run`, `Stop`, `Search Archived`, `Search Historical`, ...). This should have similar wording.

Also, I personally don't think it should be 2nd option and get in the way of the main three `Run`, `Stop`, `Delete` options.

Reply:
I think we can go with `Get Hit Count` no problem. I wanted to avoid being too wordy, but it does stand out like this.

I'm not sure why `Run` is in this menu at all considering we have a run button right next to it.
Same goes for stop since `Run` becomes `Stop` when we run a query.
It might belong first or second if those 2 are gone - I'm curious what @andrewkfiedler has to say about that.

Reply:
Definitely better to have it in both locations. Don't want to force a user to use the buttons especially in case (eventually) someone asks us to allow custom buttons there , better to have all options available in the menu

Reply:
I'm not sure that cramming another button would be worthwhile in this case; the hit count query is a special kind of query much like `Search Historical`.  We don't make buttons for those.